app/scripts/TrackRenderer.js

```diff
@@ -574,6 +585,10 @@ class TrackRenderer extends React.Component {
     this.yDomainLimits = yDomainLimits;
     this.zoomLimits = zoomLimits;
 
+    // Reset the zoomTransform, if the initial domain changed
```

Comment:
The " if the initial domain changed" is slightly confusing here as there is no `if` clause. I would just say, reset the zoom transformation on the HiGlass side to avoid future pan&zoom jumps. Or something like that.